src/sensors/bno080sensor.cpp

```diff
@@ -119,6 +120,9 @@ void BNO080Sensor::motionLoop()
             imu.getGameQuat(quaternion.x, quaternion.y, quaternion.z, quaternion.w, calibrationAccuracy);
             quaternion *= sensorOffset;
 
+            uint8 accuracy;
```

Comment:
Please use `cstdint` types.
```suggestion
            uint8_t accuracy;
```